.travis.yml

```diff
@@ -497,8 +497,9 @@ script:
     #-------------------------------------------------------------------------------
 
     # Build and execute all examples.
-    - ./travis/compileExec.sh "example/helloWorld" ./helloWorld    
-    - ./travis/compileExec.sh "example/vectorAdd" ./vectorAdd
+    - ./travis/compileExec.sh "example/copy"       ./copy        
+    - ./travis/compileExec.sh "example/helloWorld" ./helloWorld
+    - ./travis/compileExec.sh "example/vectorAdd"  ./vectorAdd
```

Comment:
Could you please remove the alignment and just use the single space in between?
alpaka does not use alignment anywhere, only indendation.